eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/beaconv2/BeaconServiceV2.java

```diff
@@ -139,7 +143,8 @@ public BeaconAlleleResponse find(String chromosome, Long start, Long startMin, L
 
     private void checkParameters(String chromosome, String referenceBases, Long start, Long end,
                                  String alternateBases, String variantType, String assemblyId,
-                                 String includeDatasetResponses) throws IllegalArgumentException {
+                                 String includeDatasetResponses, Long startMin, Long startMax, Long endMin,
```

Comment:
Fields `endMin` and `endMax` are not used in this method and they need to be checked when receiving imprecise queries